docker/test/fasttest/run.sh

```diff
@@ -253,7 +253,7 @@ function run_tests
     start_server
 
     set +e
-    time clickhouse-test --hung-check -j 8 --order=random \
+    time clickhouse-test --hung-check -j "$(nproc)" --order=random \
```

Comment:
bad idea :)

Reply:
Why do we limit it to 8? The host running fast tests has 32 CPUs on it

Reply:
It's an amount of concurrent tests. One test can run query which utilizes all cores.

Reply:
But usually don't.

For full CPU usage we can have `$(($(nproc) / 2))`
Maybe some tests will become flaky, but our flaky check should always cover most of these cases.

Reply:
I've done it as a half of CPUs with min 1